cmake/FindGeode.cmake

```diff
@@ -26,8 +26,6 @@ else()
     file(TO_CMAKE_PATH "$ENV{GEODE_ROOT}" _ENV_GEODE_ROOT)
```

Comment:
Should not be looking for an environment variable `GEODE_ROOT`. This is a CMake variable the should override `GEODE_HOME` environment variable or any well known location of Geode.